modules/loader/src/main/java/com/opengamma/strata/loader/LoaderUtils.java

```diff
@@ -265,6 +272,27 @@ private static String normalizeIfBracketed(String value) {
     }
   }
 
+  private static String normalizeIfCommaSeparated(String value) {
+    String integerPart;
+    String decimalPart;
+
+    int decimalIndex = value.indexOf(".");
+    if (decimalIndex < 0) {
+      integerPart = value;
+      decimalPart = "";
+    } else {
+      integerPart = value.substring(0, decimalIndex);
+      decimalPart = value.substring(decimalIndex);
+    }
+
+    // only remove commas in the integer part, if those commas are followed by pattern "###," or ends with pattern "###"
+    // E.g. "1,234,567" => "123456" or "1,234" => "1234"
+    // Should ignore incorrectly placed commas, e.g. "1,234,57" => "1234,57" or "1,234," => "1234,"
+    String normalizedIntegerPart = integerPart.replaceAll("(?<=\\d),(?=\\d{3}(,|$))", "");
```

Comment:
If a regex is being used, it should be a static constant, but regexes tend to be quite slow. Simply removing commas from the input string would be simpler and probably good enough.

Reply:
in your view would it acceptable to parse `1,234,56` as `123456` or should it fail?

Reply:
Seems like the simplest is 123456. Seems fine so long as it is specified in the method Javadoc